.github/ISSUE_TEMPLATE/translation_issue.yml

```diff
@@ -23,6 +23,7 @@ body:
     label: Utility with translation issue
     options:
       - General
+      - PowerToys Awake
```

Comment:
I suggest `- Awake`

Reply:
I deny this suggestiojn ,because we will rename it everywhere to PowerToys Awake